lib/rubygems/specification.rb

```diff
@@ -2102,7 +2102,7 @@ def mark_version
 
   def method_missing(sym, *a, &b) # :nodoc:
     if @specification_version > CURRENT_SPECIFICATION_VERSION and
-      sym.to_s =~ /=$/
+        sym.to_s.end_with?("=")
```

Comment:
Same here about indentation? If the second indendation is better, can we fix it separately by applying an indentation cop :)?